lib/entity/civimelder/Activiteit.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace CsrDelft\entity\civimelder;
+
+use CsrDelft\common\ContainerFacade;
+use CsrDelft\common\datatable\DataTableEntry;
+use CsrDelft\repository\civimelder\ActiviteitRepository;
+use CsrDelft\repository\civimelder\DeelnemerRepository;
+use CsrDelft\service\security\LoginService;
+use DateInterval;
+use DateTimeImmutable;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation as Serializer;
+
+/**
+ * @ORM\Entity(repositoryClass=ActiviteitRepository::class)
+ * @ORM\Table(name="civimelder_activiteit")
+ */
+class Activiteit extends ActiviteitEigenschappen implements DataTableEntry {
```

Comment:
Misschien deze of `CsrDelft\entity\groepen\Activiteit` hernoemen, voorkomt wat onduidelijkheid.